aci/resource_aci_dhcprelayp.go

```diff
@@ -81,16 +83,44 @@ func resourceAciDHCPRelayPolicy() *schema.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 						"addr": {
-							Type:         schema.TypeString,
-							Required:     true,
-							ValidateFunc: validation.IsIPv4Address,
+							Type:     schema.TypeString,
+							Required: true,
+							StateFunc: func(val interface{}) string {
+								splitVal := strings.Split(val.(string), "/")
+								if len(splitVal) <= 1 {
+									ip := net.ParseIP(val.(string))
+									return ip.String()
+								} else {
+									ip := net.ParseIP(splitVal[0])
+									return ip.String() + "/" + splitVal[1]
+								}
+							},
+							ValidateFunc: schema.SchemaValidateFunc(validateIPAddress()),
```

Comment:
Have you tried: validation.IsIPAddress ?

Reply:
Yes, replaced the custom function with the inbuilt one